slapp/transforms/transform_pipeline.py

```diff
@@ -86,8 +87,20 @@ class TransformPipelineSchema(argschema.ArgSchema):
                      "histogram adjustment"))
     webm_bitrate = argschema.fields.Str(
         required=False,
-        default="192k",
+        default="0",
         description="passed as bitrate to imageio-ffmpeg.write_frames()")
+    webm_quality = argschema.fields.Int(
+        required=False,
+        default=20,
+        description=("Governs encoded video perceptual quality. "
+                     "Can be from 0-63. Lower values mean higher quality. "
+                     "Passed as crf to ffmpeg")
+    )
+    webm_parallelization = argschema.fields.Int(
```

Comment:
Nit-ish: The default here should probably be 1. If we're running it on the cluster, you could be asking for more resources than are allocated.